<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>693</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="startwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>1200</width>
     <height>800</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#startwidget{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0.994, y2:0.971591, stop:0.215909 rgba(0, 124, 162, 255), stop:0.59322 rgba(32, 36, 109, 255), stop:0.960452 rgba(61, 86, 138, 255), stop:1 rgba(61, 86, 138, 255));
}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>371</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 28pt &quot;MV Boli&quot;; color: rgb(255, 255, 255);

</string>
    </property>
    <property name="text">
     <string>choose your account</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="list_d">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>811</width>
      <height>471</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(125, 195, 206);</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>809</width>
       <height>469</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="back_1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>630</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:7px;
background-color:rgb(191, 191, 191);
font: 14pt &quot;MV Boli&quot;;</string>
    </property>
    <property name="text">
     <string>back</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
